src/pseudopeople/utilities.py

```diff
@@ -48,30 +48,43 @@ def format_user_configuration(user_dict: Dict, default_config) -> Dict:
     """Formats the user's configuration file as necessary so it can properly
     update noising configuration to be used
     """
-    user_dict = _change_age_miswriting_perturbations_to_dict(user_dict, default_config)
+    user_dict = _format_age_miswriting_perturbations(user_dict, default_config)
 
     return user_dict
 
 
-def _change_age_miswriting_perturbations_to_dict(
-    user_dict: Dict, default_config: ConfigTree
-) -> Dict:
+def _format_age_miswriting_perturbations(user_dict: Dict, default_config: ConfigTree) -> Dict:
     # Format any age perturbation lists as a dictionary with uniform probabilites
-    # TODO: Should we allow for a user to provide a partial dictionary path to update everything at once?
-    key = "possible_perturbations"
-    for k, v in user_dict.items():
-        if isinstance(v, dict):
-            _change_age_miswriting_perturbations_to_dict(v, default_config[k])
-        elif k == key:
-            perturbations_dict = {}
-            # Replace default configuration with 0 probabilities
-            for k0 in default_config[k]:
-                perturbations_dict[k0] = 0
-            # Add user keys with uniform probabilities
-            uniform_prob = 1 / len(v)
-            for x in v:
-                perturbations_dict[x] = uniform_prob
-            user_dict[k] = perturbations_dict
+    for form in user_dict:
+        user_perturbations = (
+            user_dict[form]
+            .get("age", {})
+            .get("age_miswriting", {})
+            .get("possible_perturbations", {})
+        )
+        if not user_perturbations:
+            break
```

Comment:
This should be `continue` not `break` or else we won't check the other forms.

Reply:
Whoa - good catch